packages/app/config/webpack.common.js

```diff
@@ -63,6 +50,12 @@ const ESLINT_PLUGIN_VUE_INDEX = `module.exports = {
 
 const sepRe = `\\${path.sep}`; // path separator regex
 
+const threadPoolConfig = {
+  workers: 2,
```

Comment:
Did this make it faster than `cpu().length - 1`?

Reply:
I didn't see any obvious improvements, even when using `thread-loader` vs. not using it, TBH. Can you maybe also test without it / with different numbers of workers and see what you get?

Reply:
Didn't see improvements either, let's keep it at 2.